helm/libero-reviewer/values.yaml

```diff
@@ -7,8 +7,8 @@ replicaCount: 1
 images:
   tag: latest
   pullPolicy: Always
-  server:
-    repository: liberoadmin/reviewer_server
+  submission:
+    repository: liberoadmin/reviewer-submission
```

Comment:
... and now its hard-coded here ... would be good if we could get consistency? (if that's possible)

Reply:
is the problem the `images` key `submission`, or the value `liberoadmin/reviewer-submission`?

Reply:
I guess the motivation for all my comments on this review is that its hard to see the consistency throughout the config. Could `libero-reviewer.fullname` could be split to `libero-reviewer.submission.containerName`, libero-reviewer.submission.k8sName` and libero-reviewer.contextName` etc - rather than having parts of it stitched together in this files? :smile: It's grating my sense of symmetry - maybe it's me :wink:

Reply:
In fairness the `fullname` concept is a Helm-generated concept, not sure if it's a standard or just a default. If you look in `_helpers.tpl` you'll see there is some logic to determine it in addition to concatenation, for example the possibility (as of today unused) to override it completely.